### DISCUSSION TITLE
Grafana

### DISCUSSION THREAD

**Discussion by Julien4135** [tillsteinbach/VWsFriend#483] [Q&A]
Hi, can you help me with a problem with Grafana? I can't edit the charts at all and the edit function in the middle at the top of the chart.

![image](https://user-images.githubusercontent.com/93113346/235296119-346dd183-ab2e-4d43-8777-f26e10370275.png)

**Comment by tillsteinbach** [comment]
The charts are locked for editing because they would be overwritten by an update. If you want to make changes better copy the dashboard and edit that one.

**Comment by Julien4135** [comment]
ok but it was possible before, I had it, it is since 3.0?

**Comment by Kaputt4** [comment]
I think it has to be with some Grafana versions. I couldn't even edit my dashboards copies until yesterday's VWsFriend 0.23.10 release. Panel descriptions weren't visible either. It's fixed now, although original dashboards can't be saved as they're provisioned dashboards (but it also happened before, as @tillsteinbach says, to ensure updates).

**Reply by Julien4135** [reply]
Thanks yes with 0.23.10 it's ok ;)
